routes/actions.js

```diff
@@ -267,6 +268,7 @@ if (!config.requireInviteLinks) {
           if (error) {
             debug.error({ req, error });
           }
+          setSignupLanguage(req, res);
```

Comment:
This seems unrelated to the team feature?

Reply:
How do you want to handle this?  I could delete this PR, rollback changes, rebase and put changes in a feature branch.

Reply:
Yes, that makes sense to me.